dune.opam

```diff
@@ -1,18 +1,41 @@
-opam-version: "1.2"
+opam-version: "2.0"
 maintainer: "opensource@janestreet.com"
 authors: ["Jane Street Group, LLC <opensource@janestreet.com>"]
 homepage: "https://github.com/ocaml/dune"
 bug-reports: "https://github.com/ocaml/dune/issues"
-dev-repo: "https://github.com/ocaml/dune.git"
+dev-repo: "git+https://github.com/ocaml/dune.git"
+doc: "https://dune.readthedocs.io/en/latest/"
 license: "MIT"
+depends: [
+  "ocmal" {>= "4.02"}
```

Comment:
typo

Reply:
Fixed, thanks. No idea, how installing locally didn't catch this.